docs/en_us/internal/testing.rst

```diff
@@ -70,6 +70,12 @@ UI Acceptance Tests
    write end-user acceptance tests directly in Python, using the
    framework to maximize reliability and maintainability.
 
+Internationalization Tests
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  Any new text that is added should be internationalized and translated.
```

Comment:
I saw this as more of a summary of the document, but I'm okay removing this one section. So you're suggesting removing the top addition, but keeping the bottom addition (with the commands)?